client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -63,12 +67,26 @@ class PacmanGame extends Component {
 
   startGame = () => {
     const { config, pacman } = this.state;
-    joinGame({ playerId: uuid.v1(), ...pacman });
-    currentGameState();
+    const playerId = uuid.v1();
+    this.setState({ playerId });
+    joinGame({ playerId, pacman });
+    getCurrentGameState();
     this.animationHandler = setInterval(
       this.animateGame,
       config.refreshRate,
     );
+    setInterval(() => {
```

Comment:
This is not required. Server will automatically start the game after 3 seconds. You are getting call stack error because of improper use of setInterval.

Reply:
So how to send & receive updates? It is happening because the payload is huge in size. It works fine when small obj is sent.